src/Common/src/System/Net/SafeCloseSocket.Windows.cs

```diff
@@ -117,6 +117,8 @@ private void InnerReleaseHandle()
             }
         }
 
+        private void UnblockSocket(InnerSafeCloseSocket innerSocket) {}
```

Comment:
Please add comments to this to explain why this method is a no-op on Windows.

Reply:
Would you see harm of calling shutdown() on Windows even if it is not needed @davidsh  -> just for consistency reason? For now, I was trying to keep the change minimal as possible.

Reply:
>Would you see harm of calling shutdown() on Windows even if it is not needed

We already have an explicit API that developers can call to do Shutdown().  I don't think it is a good idea to do this automatically.

I also don't understand why these changes are needed in this PR.  It seems like it isn't clear as to the root cause.

Reply:
The difference is that on Unix calling read() or accept() will block calling thread until operation fails or succeeds ->  possibly indefinitely. Unlike Async or non-blocking, there is no timeout associated with that call. That part seems to be working different way on Windows where we are really never blocked e.g. when there is request for cancelation we can stop pending operation from our socket code.

Reply:
So, if this is a platform difference in behavior (Windows vs. Linux), then you shouldn't add any code for Windows.  But you should still add comments to this no-op method to explain. And the words above are probably what you need in your comment for source code.